src/main/java/se/liu/ida/hefquin/engine/wrappers/lpgwrapper/impl/NodeMappingToURIsImpl.java

```diff
@@ -0,0 +1,26 @@
+package se.liu.ida.hefquin.engine.wrappers.lpgwrapper.impl;
+
+import org.apache.jena.graph.Node;
+import org.apache.jena.graph.NodeFactory;
+import se.liu.ida.hefquin.engine.wrappers.lpgwrapper.data.impl.LPGNode;
+
+public class NodeMappingToURIsImpl implements NodeMapping{
+
+    protected final String NS = "https://example.org/";
+    protected final String NODE = "node/";
+
+    @Override
+    public Node mapNode(final LPGNode node) {
+        return NodeFactory.createURI(NS + NODE + node.getId());
+    }
+
+    @Override
+    public LPGNode unmapNode(final Node node) {
+        if (!node.isURI())
+            throw new IllegalArgumentException("LPG2RDF configuration only accepts URI Node mappings");
```

Comment:
Please change the message of this exception in a way similar to what I suggested for the blank-node-related case in the other class.